### PR TITLE
Add testing docs in README

### PR DESCRIPTION
**Pull request by hqhq**
Signed-off-by: Qiang Huang <h.huangqiang@huawei.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,23 @@ make
...
+### Testing:
+
+You can run test for runC by using command:
+
+```bash
+# make test
+```
+
+Note that test cases are run in Docker container, so you need to install
+`docker` first.
```
Comment (wking) [inline README.md]: The last I heard (although I haven't been following this closely) there were [minimum version requirements for Docker compatibility][1].  We might want to list any such requirements here.

[1]: https://github.com/opencontainers/runc/issues/186#issuecomment-128523088

**Comment by hqhq** [comment]
@wking Updated, thanks.

File: README.md [hunk 2]
```diff
@@ -29,6 +29,24 @@ make
 sudo make install
 ```
 
+### Testing:
+
+You can run test for runC by using command:
```
Comment (mrunalp) [inline README.md]: s/test/tests/

File: README.md [hunk 3]
```diff
@@ -29,6 +29,24 @@ make
...
+
+```bash
+# make test
+```
+
+Note that test cases are run in Docker container, so you need to install
+`docker` first. And test requires mounting cgroups inside container, it's
+done by docker now, so you need a docker with version new than 1.8.0-rc2.
```
Comment (mrunalp) [inline README.md]: ...so you need a docker version newer than..

**Comment by hqhq** [comment]
@mrunalp Fixed, thanks.

**Comment by mrunalp** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

@hqhq can you rebase please?

**Comment by hqhq** [comment]
@crosbymichael Rebased, thanks.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**